pydantic/_internal/_decorators.py

```diff
@@ -649,6 +651,7 @@ def make_generic_field_serializer(
         if n_positional == 1:
             func1 = cast(PlainSerializerWithoutInfo, serializer)
 
+            @wraps(func1)
```

Comment:
These function names come through in error messages, hence the `wraps`, we could also use clearer names for `_wrap1` etc.

Reply:
FYI the reason they are called `_wrap1`  and not all just `wrap` is that MyPy really doesn't like it if you re-define the function, even if they are never defined in the same scope